Sources/PackageLoading/ToolsVersionLoader.swift

```diff
@@ -153,11 +174,23 @@ public class ToolsVersionLoader: ToolsVersionLoaderProtocol {
         guard let version = ToolsVersion(string: versionSpecifier) else {
             throw Error.malformedToolsVersion(specifier: versionSpecifier, currentToolsVersion: currentToolsVersion)
         }
+        
+        // Ensure that for Swift ≤ 5.3, a single U+0020 is used as spacing between "//" and "swift-tools-version".
+        // Use `ToolsVersion(version:)` instead of `ToolsVeresion(string:)` here to avoid forced unwrappiing.
+        guard spacingAfterSlashes == " " || version > ToolsVersion(version: "5.3.0") else {
```

Comment:
I think there are literals for the known tool versions that could be used here, e.g. `version > .v5_3`.

Reply:
Thanks for the tip! I switched to using `version > .5_3` in 46fe6ab75764948ea49f8698aec95ec80b41b3fd.